### ISSUE TITLE
Resolve build failure

### ISSUE THREAD

**Issue by jmitchell** [dan-t/rusty-tags#3]
`term` v0.1.13 fails with 25 errors using a recent nightly (`rustc 1.0.0-nightly (809a554fc 2015-03-23) (built 2015-03-24)`). Those errors are resolved when I change the `term` version to `"*"`, but then these 3 errors pop up.

```
src/dependencies.rs:182:24: 182:34 error: the trait `std::ffi::AsOsStr` is not implemented for the type `P` [E0277]
src/dependencies.rs:182    let mut file = try!(File::open(path));
                                               ^~~~~~~~~~
<std macros>:1:1: 6:57 note: in expansion of try!
src/dependencies.rs:182:19: 182:42 note: expansion site
src/tags.rs:114:18: 114:37 error: the trait `std::ffi::AsOsStr` is not implemented for the type `P` [E0277]
src/tags.rs:114                 .open(into_tag_file)
                                 ^~~~~~~~~~~~~~~~~~~
src/tags.rs:317:8: 317:28 error: the trait `std::ffi::AsOsStr` is not implemented for the type `P` [E0277]
src/tags.rs:317    cmd.current_dir(git_dir)
                       ^~~~~~~~~~~~~~~~~~~~
error: aborting due to 3 previous errors
Could not compile `rusty-tags`.
```

I haven't delved any deeper, but I suspect recent changes to Rust may explain it.

What version are you using? The README links to a page where rust nightlies and alpha.2 are available, so it's not totally clear.

**Comment by jmitchell** [comment]
I tried building with `rustc 1.0.0-alpha.2 (522d09dfe 2015-02-19) (built 2015-02-19)` too, but the `glob` dependency fails to build.

**Comment by dan-t** [comment]
I just released rusty-tags 0.3.1 which should be buildable with 'rustc 1.0.0-nightly (123a754cb 2015-03-24) (built 2015-03-25)'.
